### PR TITLE
 javascript3 week2 

### PR DESCRIPTION
**Pull request by Darya-s**

### CODE REVIEW

File: javascript/javascript3/week2/main.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            console.log("Blue target has been moved");
+
+            moveElement(document.querySelector("li:nth-child(3)"), { x: 400, y: 20 }).then(() => {
+                console.log("Green target has been moved");
+
+            }).catch((err) => console.log('Red target went wrong'));
+
+        }).catch((err) => console.log('Blue target went wrong'));
+
+    }).catch((err) => console.log('Green target went wrong'));
```
Comment (aleks-hyf) [inline javascript/javascript3/week2/main.js:19]: Looks good, but consider simplifying this very long statement. Use variables, async/await and try/catch

File: javascript/javascript3/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+function translateOneByOne() {
+
+
+
+    moveElement(document.querySelector("li:nth-child(1)"), { x: 20, y: 300 }).then(() => {
+        console.log("Red target has been moved");
```
Comment (aleks-hyf) [inline javascript/javascript3/week2/main.js:6]: it would be good to see WHERE this target has been moved to

File: javascript/javascript3/week2/move-element.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+// Creating new scope
+{
+    const redBox = document.querySelector('ul.marks li:nth-child(1)');
```
Comment (aleks-hyf) [inline javascript/javascript3/week2/move-element.js:3]: nice use of variables

File: javascript/javascript3/week2/move-element.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    const span = document.querySelector('span');
+
+    function getRandomInt(min, max) {
+        min = Math.ceil(min);
+        max = Math.floor(max);
+        return Math.floor(Math.random() * (max - min + 1)) + min;
+    }
+
+    function randomizeStartPosition() {
+        boxes.forEach(box => {
```
Comment (aleks-hyf) [inline javascript/javascript3/week2/move-element.js:17]: nice use of forEach and array

File: javascript/javascript3/week2/resolveAfter.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+                                        } else {
+                                            reject("Failure!");
+                                        }
+                                    })
+
+                                }
+
+                                myFunction(8).then(() => {
+                                    console.log(`I am called asynchronously`); // logged out after 8 seconds
+                                });*/
```
Comment (aleks-hyf) [inline javascript/javascript3/week2/resolveAfter.html:32]: don't leave long comments in the code. git keeps track of your changes so you can get back the older version of the file any time you want.

File: javascript/javascript3/week2/resolveAfter.html [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+        async function myFunction(resolveAfter) {
```
Comment (aleks-hyf) [inline javascript/javascript3/week2/resolveAfter.html:33]: resolveAfterInSeconds is a better name

**Review by aleks-hyf — APPROVED** [review]

File: javascript/javascript3/week1/gify.html [hunk 7]
```diff
@@ -0,0 +1,30 @@
+<html>
```
Comment (aleks-hyf) [inline javascript/javascript3/week1/gify.html:1]: week1 stuff in the wrong folder. I'm only checking week2. If you want to have your week1 stuff graded, please make a separate PR with only that.

**Review by aleks-hyf — APPROVED** [review]

File: javascript/javascript3/week2/main.js (hunk 2 above)
Comment (Darya-s) [reply javascript/javascript3/week2/main.js:6]: I do not really know how to do it, and could not figure out why it is not working.

File: javascript/javascript3/week2/main.js (hunk 1 above)
Comment (Darya-s) [reply javascript/javascript3/week2/main.js:19]: I've tried to do so, commited changes.
